firstvoices/backend/serializers/fields.py

```diff
@@ -107,3 +108,30 @@ def to_internal_value(self, data):
             raise serializers.ValidationError(
                 f"Invalid value {data}. Valid values are: {', '.join(self.enum.names)}"
             )
+
+
+class RelatedTextField(serializers.ListField):
```

Comment:
Open to suggestions for a more appropriate name for this field.

Reply:
TextListField?